realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1042,10 +1169,36 @@ public double averageFloat(String fieldName) {
     // Min
 
     /**
+     * Find the minimum value of a field.
+     *
+     * @param fieldName   The field to look for a minimum on. Only int, float, and double
```

Comment:
`int` is probably referring to the storage engine's integer type. The doc should say "byte, short, int, long, float and double (boxed and unboxed)" to be complete.